docs/Makefile

```diff
@@ -49,12 +49,21 @@ help:
 	@echo "  coverage   to run coverage check of the documentation (if enabled)"
 
 clean:
-	rm -rf $(BUILDDIR)/*
+	@rm -rf $(BUILDDIR)/*
 
-html:
+clean_rst: clean
+	@rm -rf built_rst/csv/elements*
+	@rm -rf built_rst/xml/elements*
+
+rst: clean_rst
```

Comment:
By calling `clean_rst` first, this will clean out the old RST files and make new ones.